test/e2e/e2ev1.go

```diff
@@ -30,77 +29,62 @@ import (
 
 var _ = Describe("kubebuilder", func() {
 	Context("with v1 scaffolding", func() {
-		imageName := "controller:v0.0.1"
 		var testSuffix string
-		var c *config
-		var kbTest *kubebuilderTest
+		var kbc *KbContext
```

Comment:
Can we rename `KbContext` to `KBTestContext`. I think having `test` in the name helped.